src/useKeyboard.ts

```diff
@@ -27,8 +27,13 @@ export default function useKeyboard() {
     setShown(false)
     if (e) {
       setCoordinates({start: e.startCoordinates, end: e.endCoordinates})
+    } else {
+      setCoordinates({
+        start: {screenX: 0, screenY: 0, width: 0, height: 0},
```

Comment:
can we move this into an 'initialValue' do it can be used in both places (here and top of file)?
I think it would make it better.

Reply:
I would recommend doing `Object.freeze` on that object in that case, otherwise it would just take one code path somewhere in the application to be passed the dimensions and modifying them maybe as an optimisation in their function in order for the keyboard to have wrong coordinates for the rest of the lifetime of the app 😅 

```js
const emptyCoordinates = Object.freeze({ screenX: 0, screenY: 0, width: 0, height: 0 })
```

Reply:
@Gamal-Shaban could you also do this change? I think it's the last thing missing for this PR. Thanks.

Reply:
done @pvinis 
review it